third-party/README.md

```diff
@@ -0,0 +1,14 @@
+# Bareos third party libraries
+
+This is the "home" of external libraries that are used in Bareos
+
+### Adding third party libraries
+
+Each Library will be put into its own subdirectory with `git subtree` and will have its own CMake include-file (if it is not already made available by the provider of the library), that is then loaded in CMakeLists.txt
+
```

Comment:
We need information about how to handle the licenses of the third-party libraries.